### PR TITLE
Matomo introduction and how to upgrade Matomo documentation

### PR DESCRIPTION
**Pull request by MartijnVisser**
PR does a couple of things:

* Fixes a typo in the public privacy policy
* Adds introduction about Matomo
* Includes upgrade instructions for Matomo

This will close https://issues.apache.org/jira/browse/PRIVACY-13

### CODE REVIEW

**Review by bmarwell — APPROVED** [review]

**Comment by MartijnVisser** [comment]
Thanks @bmarwell - Waiting for @grobmeier to merge it (since I don't have write access to this repo)

**Comment by grobmeier** [comment]
Hi @MartijnVisser I'd like to open this repo to the privacy committee in general, I am sorry I didn't have the time to do this. I will try to do this very soon.

**Pull request merged**
